blueprint/api.apib

```diff
@@ -406,6 +279,126 @@ You can list all comments or retrieve individual comments, as well as create and
 
     + Attributes (Unprocessable Entity Response)
 
+# Group Donation Goals
+
+This resource allows a Project to raise a set amount of money(in cents, i.e. 1000 cents = $10 USD). It is one of several donation goals a project is allowed to have. A project cannot create a donation goal until they have started accepting payment with Stripe.
+
+## Donation goals [/donation-goals]
+
+### Create a donation goal [POST]
+
+To create a Donation Goal the user must be a site admin or the owner of the referenced project.
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer {token}
+
++ Response 201 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Donation Goal Response)
+
++ Response 422 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Unprocessable Entity Response)
+
+### List all donation goals for a project [GET]
+
++ Parameters
+
+    + project_id (number) - Id of a project.
+    + id (number) - The `id` of a donation goal.
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+
++ Response 200 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Donation Goals Response)
+
++ Response 404 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Record Not Found Response)
+
+## Donation Goal [/donation-goals/{id}]
+
+### Retrieve a donation goal [GET]
+
++ Request
+
+    + Headers
+
+          Accept: application/vnd.api+json
+
++ Response 200 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Donation Goal Response)
+
+### Update a donation goal [PATCH]
+
+To update a Donation Goal the user must be a site admin or the owner of the referenced project.
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer {token}
```

Comment:
You wrote a `Donation Goal Update Request` object but it's not used here at all.